src/pipelines/epidemiology/config.yaml

```diff
@@ -314,6 +314,27 @@ sources:
     test:
       metadata_query: "key.str.match('GB_.+_.+')"
 
+  # Authoritative Data sources for GB_UK
+  - name: pipelines.epidemiology.gb_authority.Covid19UkL1DataSource
+    fetch:
+      - url: "https://api.coronavirus-staging.data.gov.uk/v1/data?filters=areaType=nation&structure=%7B%22areaType%22:%22areaType%22,%22areaName%22:%22areaName%22,%22areaCode%22:%22areaCode%22,%22date%22:%22date%22,%22cumCasesByPublishDate%22:%22cumCasesByPublishDate%22,%22cumDeathsByPublishDate%22:%22cumDeathsByPublishDate%22,%22cumPillarOneTestsByPublishDate%22:%22cumPillarOneTestsByPublishDate%22%7D&format=csv"
+        opts:
+          ext: csv
+        parse:
+          query: "date >= '2020-04-01'"
```

Comment:
For the query to have any effect, it needs to be a top-level item and it does not have to be within `parse`. I.e. make it a sibling of name and fetch.